samples/net/coap_client/boards/nrf7002dk_nrf5340_cpuapp.conf

```diff
@@ -30,6 +30,8 @@ CONFIG_SETTINGS_NVS=y
 CONFIG_SHELL=y
 CONFIG_SHELL_STACK_SIZE=4096
 
+CONFIG_WIFI_CREDENTIALS_STATIC_SSID=""
```

Comment:
Do we need these? referring to our discussion in https://github.com/nrfconnect/sdk-nrf/pull/12793.

Reply:
nope, thanks! I did not notice!